### PR TITLE
EVG-14073 fix annotation note metadata styling

### PR DESCRIPTION
**Pull request by ablack12**
image

### CODE REVIEW

**Review by khelif96 — APPROVED** [review]

**Comment by khelif96** [comment]
The tests are failing can you please update this test along side this pr. https://github.com/evergreen-ci/spruce/blob/82684a550fe376bf2661be5fb74c4cd911941707/cypress/integration/project/patches.ts#L24
I think the patch name just needs to be updated

**Review by malikchaya2 — CHANGES_REQUESTED** [review]
I don't know how easy it is to accomplish this, but I think ideally we should try to leave less space between the two things displayed, and not wrap the line (closer to how it's shown on the design). I also noticed that the the time and author are flipped from what the design shows. Was that done on purpose (either seem fine to me)?
image

**Comment by ablack12** [comment]
> I don't know how easy it is to accomplish this, but I think ideally we should try to leave less space between the two things displayed, and not wrap the line (closer to how it's shown on the design). 

I tweaked it a little bit (and tweaked the ticket metadata, which also wraps pretty frequently). If the name was even _longer_ than this it could still wrap, but for the regular case this will look better (also if the name is any longer than this I'll be really impressed haha).
image

> I also noticed that the the time and author are flipped from what the design shows. Was that done on purpose (either seem fine to me)?

This was done on purpose, in order to more closely match the jira ticket metadata, and was discussed at the time.

**Comment by ablack12** [comment]
> I think the patch name just needs to be updated

@khelif96 it looks like the patch isn't on that page; I'm not sure what would've changed but? I removed it.

**Review by malikchaya2 — APPROVED** [review]

**Comment by ablack12** [comment]
https://github.com/evergreen-ci/evergreen/pull/4422 this should fix the spruce test; @khelif96 can I merge?

**Comment by khelif96** [comment]
> [evergreen-ci/evergreen#4422](https://github.com/evergreen-ci/evergreen/pull/4422) this should fix the cypress test; @khelif96 can I merge?

Can you merge the test data fixes and try to rerun the tests?

File: cypress/integration/project/patches.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ describe("Project Patches Page", () => {
     cy.visit(route);
     cy.dataCy("patches-page-title").contains("evergreen smoke test Patches");
     const patchDisplayNames = [
-      "Commit Queue Merge: No Commits Added",
       "dist",
```
Comment (khelif96) [inline cypress/integration/project/patches.ts:24]:
```suggestion
      "Commit Queue Merge:"
      "dist",
```

**Pull request merged**